### PR TITLE
sig-cluster-lifecycle: update zoom links to include 77777, re-arrange meetings

### PR DESCRIPTION
**Pull request by neolit123**
related to:
https://groups.google.com/d/msgid/kubernetes-dev/CAJ18wa-x4iFW5kg3KJkFKgPC3JXhZpFV0F-utRYeSevw8xBF%3DQ%40mail.gmail.com?utm_medium=email&utm_source=footer

commits:

```
sig-cluster-lifecycle: update all zoom links to include passcode 77777 …
088cfea
Some meetings we reusing the SIG Zoom account personal room.
Thus, created new Zoom meetings for them.

Side tasks:
- Updated all agenda documents to point to the new meeting links.
- Updated the SIG calendar invites.
```

```
sig-cluster-lifecycle: list sub-project meetings under sub-projects …
ff7c8e3
The main list of SIG CL meetings should only include the main meeting
(and the one with alternative time in China).
```

/sig cluster-lifecycle

### CODE REVIEW

**Comment by neolit123** [comment]
viewing one commit at a time is better.
/assign @castrojo @mrbobbytables

File: sigs.yaml [hunk 1]
```diff
@@ -870,98 +870,98 @@ sigs:
...
+    url: https://zoom.us/j/566930821?pwd=N2JuRWljc3hGS3ZnVlBLTk42TFlzQT09
     archive_url: http://bit.ly/k8s-capz-agenda
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
   - description: Cluster API Provider DigitalOcean office hours
     day: Thursday
     time: "09:00"
     tz: PT (Pacific Time)
     frequency: monthly, second Thursday of the month
-    url: https://zoom.us/j/91312171751?pwd=dFRsM3pvV3BKQUM0WXhyd2twZUlnZz09
+    url: https://zoom.us/j/91312171751?pwd=bndnMDdJMkhySDVncjZoR1VhdFBTZz09
```
Comment (neolit123) [inline sigs.yaml]: this one already had code.

File: sigs.yaml [hunk 2]
```diff
@@ -870,98 +870,98 @@ sigs:
...
-    url: https://zoom.us/j/612375927
+    url: https://zoom.us/j/612375927?pwd=MldxRnRSOExCVW1rbjM4ZzBSc3MvUT09
     archive_url: https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit
   - description: kops Office Hours
     day: Friday
     time: "09:00"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://zoom.us/my/k8ssigaws
+    url: https://zoom.us/j/97072789944?pwd=VVlUR3dhN2h5TEFQZHZTVVd4SnJUdz09
```
Comment (neolit123) [inline sigs.yaml]: this one used the main SIG room, but created a new meeting ID for it.

File: sigs.yaml [hunk 3]
```diff
@@ -870,98 +870,98 @@ sigs:
...
     archive_url: https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit
   - description: kubeadm Office Hours
     day: Wednesday
     time: "09:00"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://zoom.us/j/179916854
-    archive_url: https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo/edit
+    url: https://zoom.us/j/179916854?pwd=dzRhbjFnRGVQRDVUVHY1a29JV2JxUT09
+    archive_url: https://docs.google.com/document/d/1ONcoy8bOw8SWPUwXxnKeRZST3lnUxYpUv4Y6466h9Ek/edit
```
Comment (neolit123) [inline sigs.yaml]: updated agenda doc link too.

File: sigs.yaml [hunk 4]
```diff
@@ -1023,7 +1023,7 @@ sigs:
       time: "10:00"
       tz: PT (Pacific Time)
       frequency: biweekly starting Thursday June 25th, 2020
-      url: https://zoom.us/j/875399243
+      url: https://zoom.us/j/92253194848?pwd=cVVVNDMxeTl1QVJPUlpvLzNSVU1JZz09
```
Comment (neolit123) [inline sigs.yaml:987]: this ID wasn't even under the SIG account.

**Review by cpanato — APPROVED** [review]
/lgtm
/hold for others to review

File: sigs.yaml (hunk 2 above)
Comment (geojaz) [reply sigs.yaml]: thanks!

**Comment by mrbobbytables** [comment]
I'm going to lift the hold, with the amount of people working on PR'ing this stuff, things will need to rebase often if they're not merged quickly =/ 
If there is a problem, we can fix it with a follow up.
/hold cancel

**Comment by mrbobbytables** [comment]
Linking to tracking issue: https://github.com/kubernetes/community/issues/5146

**Pull request merged**